packages/@ourworldindata/grapher/src/dataTable/DataTable.tsx

```diff
@@ -571,7 +578,27 @@ export class DataTable extends React.Component<{
         ) : null
     }
 
-    render(): React.ReactElement {
+    private renderNoDataModal(): React.ReactElement {
+        return (
+            <svg
+                width={this.bounds.width}
+                height={this.bounds.height}
+                style={SVG_STYLE_PROPS}
+            >
+                <NoDataModal
+                    manager={this.manager}
+                    bounds={this.bounds}
+                    message={`No ${a(this.entityType)} matches this query`}
```

Comment:
I think indefinite isn't working here - "no **a** country"

"No ____" is always singular though, so I don't think we need this.

![image](https://github.com/user-attachments/assets/035bfb0c-c186-4625-bfb8-abcdad293a75)

Reply:
Oops, good catch!